vendor.conf

```diff
@@ -39,8 +39,9 @@ github.com/mcuadros/go-version                6d5863ca60fa6fe914b5fd43ed8533d756
 github.com/robfig/cron                        v1.1
 
 github.com/rancher/rdns-server                bf662911db6acce4d6a85d2878653f68413b9176
-github.com/rancher/norman                     196df5ed9da30dac3aa715879f44616238b8085f
-github.com/rancher/kontainer-engine           447b7766d6aa1f15986c182eb2d7b7e97ef3678e
+github.com/rancher/types                      9c5577f95855fcb7332829acf80e486b6b382934 https://github.com/aiwantaozi/types
+github.com/rancher/norman                     0557aa4ff31a3a0f007dcb1b684894f23cda390c
```

Comment:
Why are you updating norman and kontainer-engine? Does it meas you need to rebase?

Reply:
rebased